keyvault/data-plane/azure-keyvault-webkey/pom.xml

```diff
@@ -97,32 +96,16 @@
 			<plugin>
 				<groupId>org.codehaus.mojo</groupId>
 				<artifactId>build-helper-maven-plugin</artifactId>
-				<version>1.12</version>
 			</plugin>
 
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-compiler-plugin</artifactId>
```

Comment:
You removed azure-compiler-plugin from azure-keyvault-extensions, but left here, is there any specific reason for that?